content/10.blackbox.md

```diff
@@ -1,17 +1,15 @@
-The second for the most widely used DL implementations.  
-Because of the large number of parameters and non-linear relationships among features, DL models are hard to interpret when compared to other ML models. 
+In ML, a model's interpretability refers to ability to identify the discriminative features used for learning.
+ML models vary in widely in terms of interpretability, ranging from being fully transparent to being a “black-box”.
+In the former case, a clear description of the features' importance can found, such as with logistic regression or decision trees. 
+However, DL models are harder to interpret when compared to other ML models because of their large number of parameters and the non-linear relationships between their features.
 
-There are, however, many strategies to interpret a DL models.
-For example, autoencoders (AE) is a family of unsupervised methods that aim to learn a new (encoded) representation and minimize the error between the new representation and the input data. 
-Tan et al., [@doi:10.1142/9789814644730_0014] used a denoising AE to summarize key features from breast cancer dataset.
-The authors could map encoded features to clinical characteristics relevant to the disease. 
+Model interpretability is notably important in the biomedical field, where the decisions of models can mean the difference between life and death.
+Many authors attribute the lack of pervasiveness of DL tools in healthcare because of the inability to understand what these models learn [@doi:10.1109/JBHI.2016.2636665; @doi:10.1038/s41551-018-0315-x]. 
+Luckily, there are many strategies to interpret both ML and DL models.
+In fact, due to the increasing demand for interpretable models, the literature on the topic is currently growing at an exponential rate [@arxiv:2001.02522].
+Therefore, instead of recommending specific methods, we suggest consulting a recent overview.
+We particularly recommend [@url:https://christophm.github.io/interpretable-ml-book/] which is freely available and continually updated.
 
-Model transparency is notably important in the biomedical field.
-Many authors attribute the lack of pervasiveness of deep learning tools in healthcare because of the inability to understand what these models learn [@doi:10.1109/JBHI.2016.2636665; @doi:10.1038/s41551-018-0315-x]. 
-In conclusion, we encourage beginners of the DL to train in parallel a base model that is easier to interpret. 
-In case the difference in accuracy is too high to trade-off with the DL model, pre-training AE may help to dissect which are discriminative features. 
-Alternatively, algorithms based on Garson's work [@tag:garson-interpreting] can help to investigate the weights of a DL model to better understand it [**TODO detail Garson’s algorithm?**].
+In conclusion, when using DL, be sure to analyze your model to identify how its inputs are interacting as it generates its predictions.
```

Comment:
Please consider expanding on the final remark:
**In conclusion, model transparency may be critical for DL projects, so consider ML models that support feature discrimination or post hoc techniques to understand the learned features. With you will be able to analyze your model to identify how its inputs are interacting as it generates its predictions.**

Reply:
I'm trying to understand the first sentence, so could you help me understand? 

We now say earlier in the tip that there are model-agnostic and DL-specific methods for interpretation. So, when you say "so consider ML models" do you mean that the reader should consider ML and not DL models that are especially interpretable (eg decision trees)? Or is that a recommendation for which types of baseline models they should use (per tip 2)?

Sorry for my confusion

Reply:
I agree this sentence was not clear in its context.
we could simplify it with: **(...) so consider using approaches for feature discrimination to understand the learned features.**